charts/ingest/values.yaml

```diff
@@ -1,6 +1,15 @@
 # Default values for convoy ingest.
 # This is a YAML-formatted file.
 # Declare variables to be passed into your templates.
+global:
+  externalDatabase:
+    enabled: false
+
+  nativeRedis:
+    enabled: false
+
+  externalRedis:
+    enabled: false
```

Comment:
I don't understand this

Reply:
It is still the same. This is only adding it at each component chart layer. If I try to run each component with the current values.yaml, it fails because those fields do not exists.

Reply:
ps: I have updated it to match with what is in the values.yaml file in the root level of the chart.